src/renderer/components/Header.tsx

```diff
@@ -111,7 +111,7 @@ export const Header = memo(() => {
                         closeOnClick
                         closeOnMouseDown
                         borderRadius={8}
-                        label="Stop"
+                        label="Stop (Shift+F5)"
```

Comment:
I don't like that I have to use two hands to use the shortcut for stopping a chain. On my keyword, the only key further away from the two controls is F6.

I think Run, Pause, and Stop should all have dedicated keys.

Reply:
I took that from vscode but I can make them different

Reply:
Did you have specific shortcuts in mind?

Reply:
Honestly, no, not really. Maybe just F5, F6, F7?